Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN apt-get update && \
     apt-get install -y default-libmysqlclient-dev build-essential curl \
     # cmake and protobuf-compiler required for onnx install
     cmake protobuf-compiler &&  \
+    conda install python=3.6 && \
```

Comment:
`continuumio/miniconda3` uses python 3.9 by default, but for some reason `pyarrow` can't be built in Python 3.9:

https://github.com/mlflow/mlflow/runs/3157648572#step:5:761